tests/src/Kernel/Controller/ViewerControllerTest.php

```diff
@@ -121,11 +159,16 @@ public function testEditorNoViewer(): void {
       $this->assertBadRequestResponse(
         'The Collabora Online editor/viewer is not available.',
         $request,
+        $name
+      );
+      $this->assertLogMessage(
+        RfcLogLevel::WARNING,
+        "Cannot show the viewer/editor.<br>\n" . Error::DEFAULT_ERROR_MESSAGE,
         [
-          'message' => "Cannot show the viewer/editor.<br>\n" . Error::DEFAULT_ERROR_MESSAGE,
-          'level' => RfcLogLevel::WARNING,
+          '%type' => CollaboraNotAvailableException::class,
+          '@message' => '',
         ],
-        $name
+        assertion_message: $name
```

Comment:
We could add trailing commas in calls.
```suggestion
        assertion_message: $name,
```
Unfortunately it is not checked by phpcs.

Reply:
Actually we could enable it like so in phpcs.xml:
```xml
  <rule ref="SlevomatCodingStandard.Functions.RequireTrailingCommaInCall"/>
```
Maybe in this PR we should fix the code that is part of the PR, and then we add the rule in a separate issue?
After a phpcbf run I see changes in
```
        modified:   modules/collabora_online_group/tests/src/Functional/GroupMediaViewsTest.php
        modified:   modules/collabora_online_group/tests/src/Kernel/AccessTest.php
        modified:   phpcs.xml
        modified:   tests/src/Kernel/Controller/ViewerControllerTest.php
        modified:   tests/src/Kernel/CoolPreviewFormatterTest.php
        modified:   tests/src/Kernel/ViewsLinkFieldsTest.php
```

Reply:
Yes, separate PR please